apps/condo/domains/organization/schema/OrganizationEmployeeRole.js

```diff
@@ -2,14 +2,15 @@
  * Generated by `createschema organization.OrganizationEmployeeRole 'organization:Relationship:Organization:CASCADE; name:Text; statusTransitions:Json; canManageOrganization:Checkbox; canManageEmployees:Checkbox; canManageRoles:Checkbox; canManageIntegrations:Checkbox; canManageProperties:Checkbox; canManageTickets:Checkbox;' --force`
  */
 const { COUNTRY_RELATED_STATUS_TRANSITIONS } = require('@condo/domains/ticket/constants/statusTransitions')
-const { Checkbox, Virtual } = require('@keystonejs/fields')
+const { Checkbox, Virtual, Select } = require('@keystonejs/fields')
 const { LocalizedText } = require('@condo/keystone/fields')
 const { GQLListSchema } = require('@condo/keystone/schema')
 const { historical, versioned, uuided, tracked, dvAndSender } = require('@condo/keystone/plugins')
 const { Organization } = require('../utils/serverSchema')
 const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
 const access = require('@condo/domains/organization/access/OrganizationEmployeeRole')
 const get = require('lodash/get')
+const { TICKET_VISIBILITY_OPTIONS, ORGANIZATION_TICKET_VISIBILITY } = require('@condo/domains/scope/constants')
```

Comment:
it's better to move this const inside the org domain

Reply:
moved